### DISCUSSION TITLE
Initiative Group NPC

### DISCUSSION THREAD

**Discussion by dhymasriyanto** [seanmccarty/RequestedRolls#46] [Q&A] [answered]
Hi! I wonder if there is a way to group initiative rolls for too many NPCs? Because some DMs (including me) usually group them based on the order of the NPC's initiative. So it means you only need to roll one NPC that has the same group (type).

I use your extension because I want to show them the dice results (not on the default FGU feature that doesn't show the dice results in the chat box, but the default feature can group initiative rolls).

Thanks for your great work!

**Comment by seanmccarty** [comment]
To make sure I understand what you are asking for, the built-in initiative grouping doesn't work because you want to see the dice roll.

Do you want to show one roll for each NPC type and faction and have that initiative apply to all NPCs of that type and faction?

**Reply by dhymasriyanto** [reply]
Yeah something like that, when you have, say, eight Bandits, two Wolves, and say five PCs, the Bandits will take the same one initiative count, and the Wolves too, but for the PCs it depends on the outcome of their rolls. So say (8) Bandits will have 15 on initiative rolls, (2) Wolves will have 7, and the PCs their own results. So, I don't need to repeat the NPC rolls, just twice (for the Bandits and Wolves).

**Reply by dhymasriyanto** [reply]
![image](https://github.com/seanmccarty/RequestedRolls/assets/32106219/e70e6f01-500a-46c4-aa81-f350c56ee99f)


In this case these 4 NPCs need to have the same initiative count

**Reply by seanmccarty** [reply, marked as answer]
It sounds like this might function better as a standalone extension that adjusts the behavior of initiative rolled from the combat tracker. It sounds like the actual roll request pop-up is not important to this use case.

**Reply by dhymasriyanto** [reply]
Alright, thanks for your suggestion, I will try to find a suitable extension. Or, do you have any suggestion for me?

**Reply by seanmccarty** [reply]
Sorry, I don't have a suggestion.

**Reply by dhymasriyanto** [reply]
Alright, thank you!
